Assets/AWSIM/Scripts/Environments/V2I/V2I.cs

```diff
@@ -0,0 +1,69 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using GeometryUtility = AWSIM.Lanelet.GeometryUtility;
+
+namespace AWSIM
+{
+    public class V2I : MonoBehaviour
+    {
+        [SerializeField,
+        Tooltip("traffic_signals publication frequency - number of publications per second"),
+        Range(1.0f, 100.0f)]
+        int outputHz = 10;
+
+        private Vehicle egoVehicle;
```

Comment:
Currently, could you have Transform in public as a member? Then manually reference it from the inspector.
In the future, there may be more than one Vehicle class in a scene.